lib/internal/loader/singleton.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
Can you separate out the addition of this module into a separate commit? It seems to be common between this PR and the one for `import`-in-REPL, and it'll make the rest of the PR easier to review.